doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -123,7 +123,7 @@ Add the upstream repo as a "remote" and fetch it:
    git fetch ceph
 
 Fetching is a process that downloads all objects (commits, branches) that have
-been added since the last sync. These commands download all the branches from
+been added to ``ceph/ceph`` since the last sync. These commands download all the branches from
```

Comment:
Seems unnecessary to define `git fetch` here. I think we can merge these two sentences to make one ceph-specific sentence like:
`
These commands fetch all the branches and commits from ceph/ceph.git to the local git repo as remotes/ceph/$BRANCH_NAME and can be referenced as ceph/$BRANCH_NAME in local git commands.`

Reply:
Fair enough. I've made these changes.